x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

```diff
@@ -101,33 +101,40 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await PageObjects.share.clickShareTopNavButton();
       });
 
-      it('allow saving via the saved query management component popover with no query loaded', async () => {
+      it('allows saving via the saved query management component popover with no saved query loaded', async () => {
```

Comment:
These tests were copied+pasted into multiple places, so I've done the same. The diff here is the same as the diff above.